env.io-backend

```diff
@@ -127,9 +127,9 @@ IS_APPBACKENDLI=true
 # ------------------------------------
 # PN VARIABLES
 # ------------------------------------
-PN_SERVICE_ID=aPNServiceId
-PN_API_URL="http://io-pn-mock:${IO_PN_MOCK_PORT}"
-PN_API_URL_UAT="http://io-pn-mock:${IO_PN_MOCK_PORT}"
+PN_SERVICE_ID=${REQ_SPECIAL_SERVICE_ID}
+PN_API_URL="http://host.docker.internal:${IO_PN_MOCK_PORT}"
```

Comment:
isn't this useless if we use traefik?

Reply:
I mean: using docker internal host.

Reply:
i will revert this back to the container name, i think it is more clear